website/content/en/configurations/identity-access-modes/user-assigned-msi-mode.md

```diff
@@ -107,6 +107,13 @@ In AKS you can use the [User-assigned Kubelet managed identity](https://docs.mic
    az keyvault set-policy -n $KEYVAULT_NAME --certificate-permissions get --spn <USER-ASSIGNED MANAGED IDENTITY CLIENTID>
    ```
 
+   __*NOTE*__: If you are using the `objectVersionHistory` functionality, then the identity will also need secrets/list:

```

Comment:
Is this specific to Secrets?

Reply:
this was outdated, I've updated it to specify list is needed for all types of secrets